### ISSUE TITLE
Expose the 'choose' function

### ISSUE THREAD

**Issue by MorningPants** [iamstevendao/vue-tel-input#490]
My app used the Vue 2 version of vue-tel-input and used the choose function via `this.$refs.vueTelInput.choose(countryCode)` when the user selected their country in a different field.  In Vue 3, we cannot access the `choose` function now since it it is not included in defineExpose.  To retain the functionality that the Vue 2 version had, the choose function should be added to defineExpose.

**Comment by vmisiko** [comment]
yes i am finding this challenge to work with it in vue 3. I need to set the default currency dynamically.  Or  we can create a pull request fix to make dafaultCountry prop is reactive. When changed it updates the country code selected.

**Comment by vmisiko** [comment]
I have created a pull request that fixes this issue, Have tested it out. - [link](https://github.com/iamstevendao/vue-tel-input/pull/499)

**Comment by MorningPants** [comment]
That solution works! Thank you!
